### ISSUE TITLE
[REQUEST] Source On GitHub - Oxygen Builder Is The Future - But Buggy With Slow Development

### ISSUE THREAD

**Issue by maxlatitude** [soflyy/oxygen-bugs-and-features#1371]
Oxygen builder is the future and has the potential to become a core page builder for serious, high-end WordPress development. However, there are way too many bugs and development is moving too slow to compete outside the niche.

Soflyy can greatly benefit from community development by pushing as much of the source onto GitHub as possible, while maintaining a closed component / feature activation for professional, paid customers. Increased community development and adoption will greatly benefit all parties.

Figure out a way to monetize while releasing as much code as possible for community development. Oxygen is the future - need to RAD this thing!

**Comment by ylluminate** [comment]
I have to agree. I would hate to see this end up like Headway - and I feel that that is the direction it's going right now (see [an effectual postmortem here](https://wptavern.com/headway-themes-appears-to-be-dying-a-slow-death)).

I've been trying to push for Oxygen for some time now, but we keep being told "no, everyone is using Divi and while Oxygen seems to be better, it just doesn't have the community behind it." 

In my experience Divi is horrible - it's unintuitive and lacks so much that Oxygen brings to the table. 

If we could get better community involvement and security of codebase protection it would add much in the way of trust for anyone wanting to take the plunge into Oxygen.

**Comment by artbrander** [comment]
It actually seems to me like development has sped up somewhat, but because there are so many major features that Oxygen needs to compete, it's all still just playing catch up at this point and all the small stuff seems to get pushed down the priority list. What sets Oxygen apart is its appeal to people who are more capable developers, whereas Divi users are less comfortable with code. Oxygen could leverage this to great advantage by encouraging community involvement somehow.
